README.md

```diff
@@ -1 +1,56 @@
+
+## License
+
+Unless stated otherwise in the projects Repository our work is licensed under the [MIT License](https://choosealicense.com/licenses/mit/)
+
+## Code of Conduct
+This code of conduct outlines our expectations for all those who want to participate or interact with the Kord project via GitHub Issues, Discussions, Pull Requests, our Discord server or simmilar
+### How to behave
+The following behaviours are expected and requested of all community members:
+
+- Participate in an authentic and active way. In doing so, you contribute to the health and longevity of this community.
+- Exercise consideration, respect and empathy in your speech and actions. Remember, we have all been through different stages of learning when adopting technologies.
+- Refrain from demeaning, discriminatory, or harassing behaviour and speech.
+- Disagreements on things are fine, argumentative behaviour or trolling are not.
+
+### How not to behave
+- Do not perform threats of violence or use violent language directed against another person.
+- Do not make jokes of sexist, racist, homophobic, transphobic, ableist or otherwise discriminatory nature, or use language of this nature.
+- Do not post or display sexually explicit or violent material.
+- Do not post or threaten to post other people’s personally identifying information ("doxing").
+- Do not make personal insults, particularly those related to gender, sexual orientation, race, religion, or disability.
+- Do not engage in sexual attention. This includes, sexualised comments or jokes and sexual advances.
+- Do not advocate for, or encourage, any of the above behaviour.
+
+Please take into account that online communities bring together people from many different cultures and backgrounds. It's important to understand that sometimes the combination of cultural differences and online interaction can lead to misunderstandings. That is why having empathy is very important.
+
+### How to report issues
+If you want to report violations of another member and/or need assitance because of that persons behavior feel free to contact one of the Admins (schlaubi, lustiger.lurch) on [our Discord Server](https://discord.gg/6jcx5ev)
+
+### Consequences of Unacceptable Behaviour
+Unacceptable behaviour from any community member will not be tolerated. Anyone asked to stop unacceptable behaviour is expected to comply immediately. If a community member engages in unacceptable behaviour, the Kord maintainers may take any action they deem appropriate, up to and including a temporary ban or permanent expulsion from the community without warning.
+
+### License and attribution
+The license is based off of The [Citizen Code of Conduct](https://github.com/stumpsyn/policies/blob/master/citizen_code_of_conduct.md) is distributed by Stumptown Syndicate under a Creative Commons Attribution-ShareAlike license, we also took inspiration from the [JetBrains](https://github.com/jetbrains#code-of-conduct) Code of Conduct
```

Comment:
It's `based on the`, not `based off of`.

I'd replace the comma in `license, we also took` with a full stop and start a new sentence - `license. We also took`

Also, your link text is a bit weird - I'd fully link both `The Citizen Code of Conduct` and `The JetBrains Code of Conduct`. The way you've done things here is inconsistent, and the second link makes it look like you're just linking to JetBrains' company site.